core/include/edm/container.hpp

```diff
@@ -10,6 +10,8 @@
 #include <type_traits>
 
 // VecMem include(s).
+#include <thrust/pair.h>
```

Comment:
I'm confused. In the current setup where is the build taking this include from?

In any case, since you seem to be back to using [std::pair](https://en.cppreference.com/w/cpp/utility/pair), you should just remove this include.

Reply:
Right. I removed it